packages/dropdowns/package.json

```diff
@@ -14,10 +14,10 @@
     "dist"
   ],
   "scripts": {
-    "build": "../../utils/scripts/build.sh",
-    "build:demo": "../../utils/scripts/build-demo.sh",
-    "start": "../../utils/scripts/start.sh"
+    "build": "../../utils/scripts/build.sh && yarn build:types",
+    "build:types": "tsc -p tsconfig.build.json"
```

Comment:
Reduce scripting footprint by removing `build:types` and just adding to `build`.